pontoon/contributors/static/js/notifications.js

```diff
@@ -37,4 +39,24 @@ $(function () {
       Pontoon.markAllNotificationsAsRead();
     }, 1000);
   }
+
+  // Load remaining notifications
+  if ($('#server').data('hasMore')) {
```

Comment:
The jQuery `.data()` does not parse the dataset contents, so that if the attribute value is `data-has-more="false"` or any other valid JSON value that's not an empty string, this condition will evaluate as true. An explicit check might be required:

```suggestion
  if ($('#server').data('hasMore') === 'true') {
```

Reply:
_Every attempt is made to convert the attribute's string value to a JavaScript value (this includes booleans, numbers, objects, arrays, and null)._

https://api.jquery.com/data/#data-html5